app/Livewire/GlobalSearch.php

```
@@ -0,0 +1,371 @@
+    {
+        $this->search();
+    }
+
+    private function loadSearchableItems()
+    {
+        // Try to get from Redis cache first
+        $cacheKey = self::getCacheKey(auth()->user()->currentTeam()->id);
+
+        $this->allSearchableItems = Cache::remember($cacheKey, 300, function () {
+            $items = collect();
+            $team = auth()->user()->currentTeam();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$team'. (undefined)

(UnusedLocalVariable)